baseapp/baseapp.go

```diff
@@ -309,6 +310,7 @@ func (app *BaseApp) MountStores(keys ...storetypes.StoreKey) {
 // BaseApp multistore.
 func (app *BaseApp) MountKVStores(keys map[string]*storetypes.KVStoreKey) {
 	for _, key := range keys {
+		app.logger.Debug("Mounting KVStore", key)
```

Comment:
[microscopic nit]
Checked the repo and apparently most logger messages don't start with an upper case.

Reply:
good catch, will lowercase!